runtime/lua/vim/ui.lua

```diff
@@ -0,0 +1,32 @@
+local M = {}
+
+--- Prompts the user to pick a single item from a collection of entries
+---
+---@param entries table Arbitrary entries
+---@param opts table Additional options
+---     - prompt (string|nil)
+---               Text of the prompt. Defaults to `Pick one of:`
+---     - format_entry (function entry -> text)
```

Comment:
weird name. can be avoided by renaming `entries` to `items`.
```suggestion
---     - format_item (function entry -> text)
```